invoicer_project/clients/urls.py

```diff
@@ -1,13 +1,16 @@
 from django.urls import path
-from .views import home_view, ClientViewSet
+from .views import ClientViewSet, ClientRenderingViewSet
 
 clients_methods = ClientViewSet.as_view({'post': 'create', 'get': 'list'})
 clients_list = ClientViewSet.as_view({"get": "list"})
 delete_client = ClientViewSet.as_view({'delete': 'destroy'})
+render_home_client_html = ClientRenderingViewSet.as_view({'get': 'home_view'})
+render_add_client_html = ClientRenderingViewSet.as_view({'get': 'add_client_view'})
 
 urlpatterns = [
-    path("home/", home_view, name='home'),
+    path("home/", render_home_client_html, name='home'),
```

Comment:
I gess this view you could replace by TemplateView

Reply:
Done